apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -0,0 +1,152 @@
+import FontAwesomeV6Icon from '@code-dot-org/component-library/fontAwesomeV6Icon';
+import classNames from 'classnames';
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {Source} from '../../lab2/types';
+import MusicBlocklyWorkspace from '../blockly/MusicBlocklyWorkspace';
+import {BlockMode, DEFAULT_PACK} from '../constants';
+import MusicLibrary from '../player/MusicLibrary';
+import MusicPlayer from '../player/MusicPlayer';
+import Simple2Sequencer from '../player/sequencer/Simple2Sequencer';
+import {setPlayerContext, setExemplarPlaybackEvents} from '../redux/musicRedux';
+
+import moduleStyles from './ExemplarPlayer.module.scss';
+
+interface ExemplarPlayerViewProps {
+  source: Source;
+  title: string;
+  labSetPlaying: (playing: boolean) => void;
```

Comment:
nit: maybe `labPlayerSetPlaying` or `mainPlayerSetPlaying` or `toggleMainPlayer/LabPlayer`? Just for clarity; also/alternatively a quick comment here on what it's supposed to control?